orchestrator/src/handlers/price/sources/pyth.rs

```diff
@@ -16,15 +18,7 @@ impl Pyth {
         let request_url = "https://hermes.pyth.network/v2/price_feeds".to_string();
 
         let mut ticker_id: Option<String> = None;
-        let verity_client = get_verity_client();
-        let response = verity_client
-            .get(&request_url)
-            .send()
-            .await?
-            .subject
-            .text()
-            .await?
-            .to_string();
+        let response = reqwest::get(request_url).await?.text().await?;
```

Comment:
Is Verity no longer being used with the initial price_feeds fetch?

This is just a question.
We can leave this as it is, as there's an issue to adjust how often this specific fetch occurs.